cli/src/pcluster/models/imagebuilder_resources.py

```diff
@@ -115,7 +115,7 @@ def imagebuilder_image_is_building(self):
             if imagebuilder_image_status == "CREATE_IN_PROGRESS":
                 return True
             return False
-        except KeyError:
+        except (TypeError, KeyError):
```

Comment:
Approved, but I would also add a test for this condition in the imagebuilder tests

Reply:
Added